adv-rumors.lua

```diff
@@ -80,52 +84,36 @@ function addKeyword(choice, keyword)
     choice.key_word:insert('#', keyword_ptr)
 end
 
+function addKeywords(choice, keywords)
+    for i, keyword in ipairs(keywords) do
+        addKeyword(choice, keyword)
+    end
+end
+
 function addKeywordsForChoice(choice)
     local fulltext = choiceToString(choice)
 
     -- Special cases
     if string.find(fulltext, "slew") or string.find(fulltext, "slain") then
-        addKeyword(choice, 'slew')
         addKeyword(choice, 'slay')
-        addKeyword(choice, 'slain')
     end
 
     -- add a "sane" handling of you/your/me
-    if string.find(fulltext, " you ") or string.find(fulltext, " your ") then
+    if string.find(fulltext, 'you?%f[%W]') or string.find(fulltext, 'your?%f[%W]') then
```

Comment:
```suggestion
    if string.find(fulltext, 'your?%f[%W]') then
```

Reply:
the `?` after `your` makes it match `you` as well as `your`. `you?` makes it match `you` and `yo`.